src/api/Post.js

```diff
@@ -17,3 +16,34 @@ export const AxiosPost = async (data, handleNavigate) => {
     });
   }
 };
+
+export const AxiosGet = async (data, handleData) => {
+  try {
+    const headers = {
+      accept: "application/json",
+      Authorization: `Bearer ${localStorage.getItem("key")}`,
+    };
+    const res = await Axios.get(`/api/posts/${data}`, {
+      headers: headers,
```

Comment:
```suggestion
      headers
```
key,value가 이름이 동일하면 이렇게 쓸수도 있습니다.